pyproject.toml

```diff
@@ -75,6 +75,33 @@ requires = [
   skip-string-normalization = true
   skip-magic-trailing-comma = true
 
+[tool.ruff]
+  line-length = 120
+  target-version = "py37"
+  select = ["C9", "E", "F", "I", "PL", "S", "UP", "W", "YTT", "BLE", "C4", "RUF"]
+  ignore = [
+    "PLR2004", # Allow to use inplace values in comparisons instead of constants
+    "PLW2901", # Allow overwritten values on loops
+  ]
+
+  [tool.ruff.isort]
+    known-first-party = ["examples", "kornia", "test"]
+
+  [tool.ruff.mccabe]
+    max-complexity = 40
+
+  [tool.ruff.pylint]
+    max-args = 16
```

Comment:
Add https://beta.ruff.rs/docs/settings/#allow-magic-value-types here and then remove line 83 above.